common/common.go

```diff
@@ -22,16 +22,16 @@ type HTTPServerTimeouts struct {
 	Idle       time.Duration // Timeout to disconnect idle client connections. None if 0.
 }
 
-type BlockBuilderStatusCode uint8
+type BlockBuilderStatus uint8
```

Comment:
Normalise the use of "BlockBuilder" vs "Builder". I would just use "Builder" everywhere :)

Reply:
ah good point. only problem is that the existing implementation uses both: e.g., we hve the block builder table: https://github.com/flashbots/mev-boost-relay/blob/52eb5d81fcdeb646a61859097fee37a2b6ce4eea/database/database.go#L42-L46. i will make all of my changes use just `Builder` except when interacting with that table to provide consistency.

Reply:
https://github.com/michaelneuder/mev-boost-relay/pull/1/commits/6bb8bf357d441c7228c431d16c8dc70027748fa8